lib/dynflow/action/suspended.rb

```diff
@@ -9,16 +9,19 @@ class Action::Suspended
     end
 
     attr_reader :execution_plan_id, :step_id
-    attr_reader :wakeup_info
 
     def initialize(action)
       @execution_plan_id = action.execution_plan_id
       @step_id           = action.run_step_id
```

Comment:
How about we always set the wakeup info to start as an empty array? It would simplify the rest of the code a bit by not having to check if it is nil or not

Reply:
Not relevant anymore, I've changed the implementation.